README.md

```diff
@@ -48,6 +48,28 @@ Then follow the instructions displayed by the virtual machine.
 
 Sass files are currently generated on each pageload, just like any other template. You do not need to manually compile the Sass.
 
+## Run without Vagrant
+
+You can also run it locally without using Vagrant. First [install Foreman globally](https://github.com/ddollar/foreman#installation):
+
+```bash
+$ gem install foreman
```

Comment:
I think it's best to only put a `$` at the start of the line if you're also including the output. That's also consistent with the rest of the file. I learnt about this rule from [markdownlint](https://github.com/mivok/markdownlint/blob/25081c5b1a1312a111d22791197e6e2ad8092a0c/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output), but those docs also link through to [the markdown style guide](http://www.cirosantilli.com/markdown-style-guide/#dollar-signs-in-shell-code).